readme.md

```diff
@@ -4,9 +4,9 @@
 
 
 
-## The latest and greatest boilerplate for Infinite Red opinions
+## The latest and greatest boilerplate for our opinions
 
-This is the boilerplate that [Infinite Red](https://infinite.red) uses as a way to test bleeding-edge changes to our React Native stack.
```

Comment:
Ok I rephrased this so it mentions the initial project it was based on.  I also changed the React Native piece to read React Web.